test/TestImportTasks.py

```diff
@@ -0,0 +1,73 @@
+"""Calling 'import_task' from multiple files in different directories."""
+import os
+import shutil
+import tempfile
+import textwrap
+
+import pytest
+
+from ansiblelint.runner import Runner
+
+LAYOUT = {
+    'directories': ['tasks', 'tasks/subtasks'],
+    'files': {
+        'main.yml': textwrap.dedent(
+            """\
+            ---
+            - hosts: target
+              gather_facts: false
+              tasks:
+                - name: from main import task 1
+                  import_tasks: tasks/task_1.yml
+            """
+        ),
+        'tasks/task_1.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from task 1 import task 2
+              import_tasks: tasks/task_2.yml
+            """
+        ),
+        'tasks/task_2.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from task 2 import subtask 1
+              import_tasks: tasks/subtasks/subtask_1.yml
+            """
+        ),
+        'tasks/subtasks/subtask_1.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from subtask 1 import subtask 2
+              import_tasks: tasks/subtasks/subtask_2.yml
+            """
+        ),
+        'tasks/subtasks/subtask_2.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from subtask 2 do something
+              debug:
+                msg: |
+                  Something...
+            """
+        ),
+    },
+}
+
+
+@pytest.mark.xfail
```

Comment:
Every xfail needs a reason which should at least link some clear explanation and/or have words explaining that in place.
```suggestion
@pytest.mark.xfail(reason='https://github.com/ansible-community/ansible-lint/issues/<a_dedicated_issue_explaining_this_specific_problem>')
```